src/network/core/address.cpp

```diff
@@ -326,13 +326,10 @@ static SOCKET ListenLoopProc(addrinfo *runp)
 		DEBUG(net, 3, "[%s] setting TCP_NODELAY failed for port %s", type, address.c_str());
 	}
 
-	int on = 1;
-	/* The (const char*) cast is needed for windows!! */
-	if (setsockopt(sock, SOL_SOCKET, SO_REUSEADDR, (const char*)&on, sizeof(on)) == -1) {
-		DEBUG(net, 3, "[%s] could not set reusable %s sockets for port %s: %s", type, family, address.c_str(), NetworkError::GetLast().AsString());
-	}
+	if (!SetReusePort(sock)) DEBUG(net, 0, "[%s] setting reuse-port mode failed for %s port %s", type, family, address.c_str());
```

Comment:
Since you're touching this, the second %s says "IPvX" and the last %s gives you "IP:port (IPvX)" so that's duplicate information. Similarly "IP:port" is not really a port, is it? And is it wanted that the error from the OS is not passed to the "user" in this situation, as that might be the only thing that can help us in any way.

Reply:
To not bloat this PR any more than I have to, #9230 :)